src/main/java/com/dezorganizacja/bridge/Bridge.java

```diff
@@ -11,6 +12,8 @@ public static void main(String[] args) {
         MainPresenter presenter = new MainPresenter(message);
 
         View view = new GraphicalView();
+        View view1 = new TextView();
```

Comment:
zdecydowanie nie taka nazwa zmiennej. Trzeba zmienić na powiedzmy textView i wtedy to wyżej na np graphicalView